app/views/content/train-to-be-a-teacher/what-is-a-pgce.md

```diff
@@ -41,7 +41,7 @@ If you have a degree and are interested in gaining a qualification in education,
 
 You can get a postgraduate certificate in education through teacher training.
 
-You do not need a PGCE to be a qualified teacher in England, but it can help you develop your teaching skills and knowledge.
+You do not need a PGCE to be a qualified teacher in England, but it can help you develop your [teaching skills and knowledge](/life-as-a-teacher/teaching-as-a-career/skills-to-teach).
```

Comment:
I think this link is OK but are we implicitly implying that QTS only won't give you those skills?

Could we mitigate this by adding a skills link on the QTS page too?